packages/streams/serialport.js

```diff
@@ -117,7 +119,7 @@ SerialStream.prototype.start = function () {
         `Connected to ${this.options.device}`
       )
       this.isFirstError = true
-      const parser = new SerialPort.parsers.Readline()
+      const parser = new ReadlineParser({ delimiter: '\r\n' })
```

Comment:
It looks to me that v9 Readline's [default delimiter](https://github.com/serialport/node-serialport/blob/v9.2.8/packages/parser-readline/lib/index.js#L17) is `\n` and it is [the same in v11](https://github.com/serialport/node-serialport/blob/master/packages/parser-readline/lib/index.ts#L20), so I don't think we should change this.

While `\r\n` is what NMEA0183 specifies the serialport can be used for non-1083 data and we should not change the behavior. Not sure what `\n` delimited input would result in with this change.

Reply:
I took the liberty of changing the PR to use the default delimiter and fixed a few issues with it.